mypy/nodes.py

```diff
@@ -390,6 +405,10 @@ def set_line(self, target: Union[Token, Node, int]) -> Node:
             self.initialization_statement.set_line(self.line)
             self.initialization_statement.lvalues[0].set_line(self.line)
 
+    def set_column(self, target: Union[Token, Node, int]) -> Node:
+        super().set_column(target)
+        return self
```

Comment:
perhaps this should be removed for now (until I figure out (in FLUP?) how the delegation should work